package.json

```diff
@@ -13,6 +13,6 @@
         "lerna": "^6.6.2"
```

Comment:
The latest lerna version would be v7.1.1. Here's the release notes with breaking changes notes: https://github.com/lerna/lerna/releases/tag/7.0.0
The upgrade is pretty easy, you can just install the latest version and execute `lerna repair` which then migrates the config.

Additionally, they got rid of the `bootstrap` command, so you can just use `npm install` instead.

Here is the commit where I upgraded the frontend-configs: https://github.com/certifaction/frontend-configs/commit/fbe3d336a07ea77680901a287da1639407f3d052